docs/Controller Guide.md

```diff
@@ -0,0 +1,100 @@
+
+The **IS-04 Registration API** is exposed by the Registry. When a Node first joins the network, it uses [DNS-SD](https://specs.amwa.tv/info-004/) to discover the URL of the Registration API. It then uses the Registration API to register information about itself and all its sub-resources (Devices, Senders, Receivers, Sources and Flows). Each Node then communicates with the Registration API with a regular heatbeat message to let the Registry know that it is still on the network. Nodes also use the Registration API to update the information about themselves in the Registry whenever something changes. 
+
+The **IS-04 Query API** is also exposed by the Registry. Controllers discover the URL of the Query API using DNS-SD and then obtain an up-to-date list of all registered Nodes and sub-resources (Devices, Senders, Receivers, Sources and Flows) so that they can indicate to the user the currently available resources on the network. Controllers also use the Query API to set up websockets subscriptions to resources, so that they obtain updates every time something changes in the Registry.
+
+The **IS-04 Node API** is exposed by Nodes. It is used by other Nodes to discover information about a Node directly when there is no Registry available on the network (in peer-to-peer mode).
+
+![IS-04 Interactions in a Networked Media System](./images/IS-04_Interactions.svg)
+
+The IS-04 specification describes the requirements for Controllers in the document entitled [Controllers](https://github.com/AMWA-TV/is-04/blob/nmos-controller/docs/Controllers.md) (link to be updated after IS-04 PR accepted).
+
+### IS-05 Device Connection Management
+> *Controllers use the **IS-05 Connection API** to make connections between Senders and Receivers.*
+
+Nodes on a media network may contain one or more Devices, each of which may include any number of Senders and Receivers. The IS-05 specification describes the mechanism for making connections between Senders and Receivers.
+
+The **IS-05 Connection API** is exposed by NMOS Devices. Controllers discover the URL of the Connection API through the list of `controls` for the Device. This is part of the information that will have been registered to the IS-04 Registry by the Device's parent Node. It is available to the Controller by querying the Registry using the IS-04 Query API and examining the information returned about the Device in question. 
+
+Controllers make connections between Senders and Receivers by making calls to the Connection APIs of their parent Devices. A Controller first calls the Connection API for the Sender to update its transport parameters as required and then obtain its transport file. It then calls the Connection API for the Receiver to provide it with the transport file and complete the connection.
```

Comment:
If the example is meant to showcase RTP maybe we should mention that it is an RTP example, since WebSocket and MQTT have no transport file?
Otherwise, we probably need to make it more generic or cover other use cases potentially.

Reply:
Good point. I've added words to make it clear that it's an example for an RTP-based connection.